packages/io/src/IO.ts

```diff
@@ -303,6 +305,43 @@ export class IO {
         fs.closeSync(fs.openSync(file, "w"));
     }
 
+    /**
+     * Set file access permissions so that only the current user will have access to the file.
+     * On windows, it means "full control" for the current user.
+     * On posix, it means read & write access for the current user.
+     * Obviously, the current user must have permission to change permissions
+     * on the specified file.
+     *
+     * @param  {string} fileName - file name for which we modify access.
+     * @throws An ImperativeError when the operation fails.
+     */
+    public static giveAccessOnlyToOwner(fileName: string) {
```

Comment:
what if the file was previously an executable file?

Reply:
I modified the logic to retain the executable bit, if the owner originally had it.